_data/people/StephaneEpardaud.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Stephane Epardaud
+photo:
+level: 1
+gravatar_hash: 36671d7b88823c218c135c39f8b35c3f
+active: true
+website: https://stephane.epardaud.fr
+twitter: http://twitter.com/unfromage
+---
+Member of the Vert.x and Quarkus teams.
+
+From deep into the Nice mountains, Stéphane works for Red Hat on the Vert.x and Quarkus project.
+
+Passionate hacker in Java, C, Perl or Scheme. A web standards and database enthusiast, he implemented among other things a WYSIWYG XML editor, a multi-threading library in C, a mobile-agent language in Scheme (compiler and virtual machines), and some Web 2.0 RESTful services and rich web interfaces with JavaScript and HTML 5.
```

Comment:
no mention of Ceylon !?  Very cool about Scheme.

Reply:
Huh, I guess I didn't update my bio properly the last time I edited it. Good catch.

Reply:
Ah, it's actually listed below, pfew :)